.changeset/twenty-worms-provide.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-scaffolder-backend': minor
```

Comment:
Wanna try and make this a `patch` release if possible, see below comment.

Reply:
any reason to worry about that? isn't a new part of API surface / new feature so it would be minor?

happy to change, though if you think needed

Reply:
No, we'd ideally not want to create a breaking change here just in case people already use the exported `runComand` function. Which some of the extra `modules` do which is why the build is failing. Arguably this function shouldn't really be exported from this package either, it's a bit strange, but we can move it out later I think.

Reply:
Whilst the `scaffolder-backend` package is `0.x.x` minor is equivalent to major, so that when the package is ready for v1 we're not on version `153.0.0` :)